### ISSUE TITLE
argument `output` or `type` in `connected_habitat`

### ISSUE THREAD

**Issue by bniebuhr** [ConScape/ConScapeR#3]
Hi,

I suggest we add an argument `output` or `type` to define the type of output of the `connected_habitat()` function.
The default could be `output = "matrix"`, and the alternative `output = "raster"`.
Another option would be a `raster` parameter which is `FALSE` by default and takes `TRUE` if we want the raster.

**Comment by ninbrm** [comment]
+ I agree that this would be more user-friendly, and makes sense to do. 
- However, to allow for a raster output, you need to have the raster specification as an argument (which is not the case on the Julia side of these functions)
- Hence, I would prefer to add this on the Julia side first, and keep the R library as much as possible simply a wrapper. This is why we introduced the `mat2rast` function, which for now only exists on the R side.

In summary, the mat2rast function is a patch to allow for raster output. In my opinion, the ConScape.jl library needs a revision to integrate the Rasters.jl library and allow for better integration with raster outputs in both libraries.

**Comment by bniebuhr** [comment]
Ok, I understand your reasoning - the idea to deal with that in the Julia code side instead of in ConScapeR.
That way there is only one place to keep active development/updates, and not two...
Closing this for now then.

**Comment by bniebuhr** [comment]
Correcting here and closing here as "not planned" in the ConScapeR side, but directly within ConScape.
